src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-accent;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
```

Comment:
The variable `$border-color` is used here, but it's not defined within this file. Ensure that it is defined in a variables file that is imported, or define it here if it's missing.